src/auth/auth.controller.ts

```diff
@@ -9,13 +9,20 @@ import {
   ApiCreatedResponse,
   ApiBadRequestResponse,
   ApiOperation,
+  ApiUnauthorizedResponse, // 추가
 } from '@nestjs/swagger';
 
 import { signupSwagger, loginSwagger } from './auth.swagger';
 
 @ApiTags('auth')
 @Controller('auth')
 export class AuthController {
+  private readonly cookieOptions = {
```

Comment:
상수는 일반적으로 UPPER_SNAKE_CASE를 사용합니다. 원시값이 아니고 객체여도 생명주기동안 항상 같은 값을 가지고 있는 경우

```suggestion
  private readonly COOKIE_OPTIONS = {
```